### DISCUSSION TITLE
How to decide between GetX and GetBuilder? I read the docs, still can't draw a line, please help!

### DISCUSSION THREAD

**Discussion by ad-kmt** [jonataslaw/getx#1478] [Q&A]
Hi! I'm new to flutter development, working on an e-commerce app. I chose GetX for state management because of its popularity and reviews. I've read the docs thoroughly but still can't decide between Reactive State Manager and Simple State Manager. Is there a way to draw a line between the use-cases of both the state managers?

Also, as it is mentioned in the documentation that "There are other situations where reactive programming is really interesting, and not working with it is the same as reinventing the wheel." Can someone explain more about these situations, with some real-life examples?

@jonataslaw  Please help _/ \_

**Comment by jonataslaw** [comment]
There is no better or worse at this point. Do you feel comfortable with the observable paradigm? If so, GetX, otherwise, GetBuilder. GetBuilder gives you control over when to update, GetX will update when an observable changes.

**Reply by ad-kmt** [reply]
Thanks!
